lib/index.js

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+require('bedrock-account');
+const assert = require('assert-plus');
+const bedrock = require('bedrock');
+const bcrypt = require('bcrypt');
+const {callbackify: brCallbackify} = bedrock.util;
+const {config} = bedrock;
+const crypto = require('crypto');
+const database = require('bedrock-mongodb');
+const brPermission = require('bedrock-permission');
+const {promisify} = require('util');
+const brPermissionCheck = promisify(brPermission.checkPermission);
+const randomBytes = promisify(crypto.randomBytes);
+const {BedrockError} = bedrock.util;
+
+// load config defaults
+require('./config');
+
+// module API
+const api = {};
+module.exports = api;
+
+const PERMISSIONS = bedrock.config.permission.permissions;
+const TOKEN_TYPES = ['password', 'nonce', 'challenge'];
+// base58 alphabet; used to improve human readability
+const ALPHABET = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz';
+
+const logger = bedrock.loggers.get('app').child('bedrock-authn-token');
```

Comment:
what are we doing about namespacing log stuff?  Adding `bedrock` everywhere seems redundant.  What else uses the bedrock logging system but bedrock modules?

Reply:
We decided to prune these (if necessary) in the internal logger code in the future.